src/plugins/content/overlay/overlay.js

```diff
@@ -75,11 +75,7 @@ export default pluginFactory({
      * Enable the overlay
      */
     enable: function enable() {
-        var testRunner = this.getTestRunner();
-        var testData = testRunner.getTestData() || {};
-        var testConfig = testData.config || {};
-        var pluginsConfig = testConfig.plugins || {};
-        var overlayConfig = pluginsConfig.overlay || {};
+        var overlayConfig = this.getConfig().overlay || {};
```

Comment:
`this.getConfig()` returns with the overlay plugin config, so `.overlay` should not be called on it. BTW it can be a `const`.